common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -181,6 +187,41 @@ private static boolean tryReloadApiUrls(boolean async) {
         return true;
     }
 
+    /**
+     * Request all online players to WynnAPI
+     *
+     * @return a {@link HashMap} who the key is the server and the value is an array containing all
+     *     players on it
+     * @throws IOException thrown by URLConnection
+     */
+    public static HashMap<String, List<String>> getOnlinePlayers() throws IOException {
+        if (apiUrls == null) return new HashMap<>();
+
+        URLConnection st = new URL(apiUrls.get("OnlinePlayers")).openConnection();
+        st.setRequestProperty(
+                "User-Agent",
+                "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10.4; en-US; rv:1.9.2.2) Gecko/20100316"
+                        + " Firefox/3.6.2");
+        // st.setRequestProperty("apikey", apiUrls.get("WynnApiKey")); API key rate limits
+        // constantly, perhaps we don't need it?
```

Comment:
What does this comment mean?

Reply:
Api key rate limits constantly because it is shared between all wynntils users, but if we dont use  it then our ip has a separate rate limit

Reply:
iirc there's a comment in the discord which explains the usage of the api key for wynntils clients. I think its client specific too

Reply:
Not sure, with the Api key it always seemed rate limited, with code 429 (too many requests)